bqskit/runtime/base.py

```diff
@@ -41,15 +41,22 @@ def __init__(
         conn: Connection,
         total_workers: int,
         process: Process | None = None,
-        num_tasks: int = 0,
     ) -> None:
         """Construct an employee with all resources idle."""
         self.conn: Connection = conn
         self.total_workers = total_workers
         self.process = process
-        self.num_tasks = num_tasks
```

Comment:
Why do we not set the number of tasks anymore?

Reply:
RuntimeEmployee objects are constructed only at startup before any tasks are assigned. It was cluttering the spec in the past.